MessageCenter3/Models/Message/GroupMessageRequestModel.cs

```diff
@@ -5,10 +5,9 @@
 
 namespace MessageCenter3.Models
 {
-    public class MessageRequestModel
-    {
-        public int SenderId { get; set; }
-        public int RecipientId { get; set; }
+    public class GroupMessageRequestModel
+    {   
+        public int GroupId { get; set; }
         public int MessageType { get; set; }
```

Comment:
Do we need an enum for MassageType?

Reply:
Let's leave it here for the time being. It it finds no use, I'll remove it.